packages/auth/src/index.js

```diff
@@ -36,6 +55,30 @@ module.exports = {
   auth: {
     buildAuthMiddleware: authenticated,
     passport,
+    middlewares: {
+      google,
+    },
   },
-  constants: require("./constants"),
+  constants,
```

Comment:
This looks like it might be a partial merge artifact here, looks like all the old exports have re-appeared and the constants export has been overwritten.

Reply:
👍 all good - I think I might have merged the other way. I'll get your import list and push again